### PR TITLE
Issue 1946

### PR DESCRIPTION
**Pull request by FilipLeitner**
@DailisLangovskis  could you please add Latvian translations, thanks

### CODE REVIEW

**Comment by DailisLangovskis** [comment]
Sure

FilipLeitner ***@***.***> (šajā datumā: otrd., 2021. g. 22.
jūn. 14:42) rakstīja:

File: projects/hslayers/src/components/add-data/vector/add-data-vector.component.ts [hunk 1]
```diff
@@ -103,6 +103,8 @@ export class HsAddDataVectorComponent {
         workspace: this.hsCommonEndpointsService.endpoints.filter(
           (ep) => ep.type == 'layman'
         )[0]?.user,
+        queryCapabilities:
+          this.url && !['geojson', 'kml'].some((ext) => this.url.endsWith(ext)),
```
Comment (jmacura) [inline projects/hslayers/src/components/add-data/vector/add-data-vector.component.ts:107]: GeoJSON can have .json extension as well, so perhaps
```suggestion
          this.url && !['json', 'kml'].some((ext) => this.url.endsWith(ext)),
```
?

File: projects/hslayers/src/assets/locales/en.json [hunk 2]
```diff
@@ -31,7 +31,9 @@
     "file": "File",
     "saveToDatabase": "Save to database",
     "URL": {
-      "selectAll": "Select all Layers"
+      "selectAll": "Select all Layers",
+      "submitLayerTitle": "Submit layer title",
+      "submitLayerName": "Submit layer name"
```
Comment (jmacura) [inline projects/hslayers/src/assets/locales/en.json:36]: Why "submit"? Is it submitted somewhere? Shouldn't "fill in" be a better wording?

**Review by jmacura — APPROVED** [review]
Works nice. I left some comments.

File: projects/hslayers/src/assets/locales/en.json (hunk 2 above)
Comment (FilipLeitner) [reply projects/hslayers/src/assets/locales/en.json:36]: First thing that came tom my overheated mind. Ill redo these, thanks for pointing it out

**Pull request merged**